internal/report/report.go

```diff
@@ -16,6 +16,7 @@ type TestResult struct {
 	Artifacts  []Artifact    `json:"-"`
 	Origin     string        `json:"origin"`
 	Attempts   int           `json:"attempts"`
+	IsRDC      bool          `json:"-"`
```

Comment:
I think it's a bit more idiomatic to leave boolean verbial prefixes (`has`, `is` etc.) for methods:
```suggestion
	RDC      bool          `json:"-"`
```